test/testdrive/timestamps-multi.td

```diff
@@ -32,7 +32,7 @@ $ set schema={
 $ kafka-create-topic topic=data-consistency
 
 $ kafka-ingest format=bytes topic=data-consistency timestamp=1
-dummy,1,0,0,0
+"dummy,1,0,0,0"
```

Comment:
This seems like a weird way to hack around the JSON parsing above. If we don't do that, then we can avoid all these quotes here and below.

Reply:
In the previous testdrive review, you had suggested using `serde_json::Deserializer::from_str(row).into_iter<serde_json::Value>()` in order to split the key part of a row from the value part of a row without splitting along a special character. 

Without the "" around the byte string, serde_json::Deserializer will hit an `error: parsing json string datum: expected ident at ...`

Would you want `serde_json::Deserializer` only to be used in splitting the key from the value when both key_schema and value_schema are Avro? And then we split the key along a special character when the key_schema and/or value_schema are Bytes?

Reply:
Blergh, ok, gotcha. A better solution doesn't immediately present itself, so let's merge this as is.